### PR TITLE
Feature/dataset detail page

### PR DESCRIPTION
**Pull request by geeeeemoney**
### Description

This PR adds a dataset detail page for any dataset that is clicked (currently just in the card view, not network graph).  

dataset-detail-page

### Checklist

- [x] This PR contains tests (at least one (non-snapshot) test)

### CODE REVIEW

File: src/components/DatasetDetailPage.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+      name,
+      description,
+      tags = [],
+      updatedAt
+    } = dataset
+
+    let {
+      fields
+    } = dataset
+
```
Comment (mell0kat) [inline src/components/DatasetDetailPage.tsx:93]: What happens if fields is empty array?

File: src/components/DatasetDetailPage.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                          <InfoIcon color='disabled' fontSize='small' />
+                        </div>
+                      </Tooltip>
+                    </TableCell>
+                    )
+                  })
+                }
+              </TableRow>
+            </TableHead>
+            {/* <TableBody>
```
Comment (mell0kat) [inline src/components/DatasetDetailPage.tsx:160]: Nice!  Will be easy to add the first ten rows when we get them!

File: src/components/AppBar.tsx [hunk 3]
```diff
@@ -30,12 +38,15 @@ interface IProps extends IWithStyles<typeof styles> {}
...
 const MyAppBar = (props: IProps): ReactElement => {
   const { classes } = props
+  const { link } = classes
   return (
     <AppBar position='fixed' className={classes.appBar}>
       <Toolbar>
-        <Typography className={classes.icon} variant='h4' color='inherit' noWrap>
-          MARQUEZ
-        </Typography>
+        <Link className={link} to="/">
```
Comment (mell0kat) [inline src/components/AppBar.tsx:45]: Very good idea. Much more user friendly.

File: src/__tests__/components/JobDetailPage.test.tsx [hunk 4]
```diff
@@ -70,7 +70,7 @@ describe('JobDetailPage Component', () => {
             .text()
         ).toContain(job.name)
       })
-      it('job name should contain a link to the job description', () => {
+      it('job name should contain a link to the job location', () => {
```
Comment (mell0kat) [inline src/__tests__/components/JobDetailPage.test.tsx:73]: Good catch :)

**Review by mell0kat — APPROVED** [review]
Looks great @grantdfoster

**Review by jhubley — APPROVED** [review]
looks good!

File: src/components/DatasetDetailPage.tsx (hunk 1 above)
Comment (grantfosterdev) [reply src/components/DatasetDetailPage.tsx:93]: there's no header, it's just an empty "Paper" component box

**Pull request merged**